### ISSUE TITLE
[Due for payment 2026-04-03] [$500] Screen Reader: Global: The availability of displayed search suggestions is not announced

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77390]
## Action Performed:
Precondition: User is logged in

1. Using VoiceOver on iOS or TalkBack on Android, open the app
2. Swipe right to focus the Account tab and double-tap to activate it
3. Swipe right to focus the Profile button and double-tap to activate it
4. Swipe right to focus the Status menu item and double-tap to activate it
5. Swipe right to focus the Vacation delegate menu item and double-tap to activate it
6. Note if the available list of suggestions is announced
7. Swipe right to focus the text field and enter any search query
8. After loading is complete, note if the search result suggestions are announced

This is a global issue affecting all screen with text fields and search result suggestions
Examples:
1. Search
2. Group Chat: Start Chat
3. Profile: Country
4. Profile: Timezone
5. Wallet: Add Bank Account
6. Reports
7. Create Workspace > Company Cards: Add Card

## Expected Result:
When search suggestions appear, both initial suggestions (before typing) and resulting suggestions (after entering a query), screen readers should announce that there are available suggestions. Examples:
"Search suggestions available. 12 results".

This announcement should occur automatically, shortly after the user has finished typing and loading is complete, without requiring the user to explore the screen manually.

## Actual Result:
Before typing, the suggestions list is visually shown but not announced.
After entering a search query, new results appear, but screen readers provide no status update or hint that results are available.

Users must discover suggestions manually by swiping, which may give the impression that no results exist.

## Area issue was found in:
Profile > Status

## Failed WCAG checkpoints
WCAG 4.1.3 Status Messages
https://www.w3.org/WAI/WCAG22/Understanding/status-messages

## User impact:
When appropriate roles or properties are assigned to status messages, the new content is spoken by screen readers in such a way as to assist blind and low vision users. Assigning proper roles or properties to status messages provides possible future uses and personalization opportunities, such as the potential to be exploited by assistive technologies created for users with some cognitive disabilities.

## Suggested resolution:
Use accessibilityLiveRegion="polite" on Android and AccessibilityInfo.announceForAccessibility on iOS to notify screen readers whenever the suggestions list changes to ensure they announce them, for example:
- 'Search suggestions available. 12 results'
https://appt.org/en/docs/react-native/samples/accessibility-live-region
https://reactnative.dev/docs/accessibility#accessibilityliveregion-android
https://reactnative.dev/docs/accessibilityinfo#announceforaccessibility

## Environment(s)
Apple iPhone 14 Pro - iOS 18.7.1, Wi-Fi - Native (No mobile browser)
Samsung Galaxy Note10 Plus - Android 12, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos
<img width="1179" height="2438" alt="Image" src="https://github.com/user-attachments/assets/713612f1-6320-47d0-b224-e1fb0f6021a4" />

https://github.com/user-attachments/assets/c5542b69-b661-4d8c-ab78-cd33b19c8d15

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7006896
**Issue reported by:** NA
**Slack conversation:** NA

<details><summary>Issue Owner</summary>Current Issue Owner: @mallenexpensify</details>
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~022032270284819723181</li>
        <li>Upwork Job ID: 2032270284819723181</li>
        <li>Last Price Increase: 2026-03-13</li>
<li>Automatic offers: </li>
<ul>
<li>aimane-chnaif | Contributor | 110767844</li>
</ul></li>
    </ul>
</details>

**Comment by rushatgabhane** [comment]
@FitseTLT please disregard previous proposals

**Comment by rushatgabhane** [comment]
Awaiting proposals

**Comment by rushatgabhane** [comment]
https://expensify.enterprise.slack.com/archives/C02NK2DQWUX/p1773366763112179?thread_ts=1773366763.112179&cid=C02NK2DQWUX

assigning @aimane-chnaif and @ShridharGoel

**Comment by ShridharGoel** [comment]
@mallenexpensify This is due for payment

**Comment by marufsharifi** [comment]
@mallenexpensify, I have a question about this issue.

I was initially assigned/hired, had already opened a [PR](https://github.com/Expensify/App/pull/83096), tested, and requested review of the PR before it was reassigned due to the deadline and estimation concerns here:

https://expensify.slack.com/archives/C01GTK53T8Q/p1772739593580499


pr colosed based on this [comment](https://github.com/Expensify/App/pull/83096#issuecomment-4007378213).

Am I still eligible for compensation in this case? This is a similar case to [here](https://github.com/Expensify/App/issues/77490#issuecomment-4237764927).

Thanks in advance for your guidance!

**Comment by mallenexpensify** [comment]
👀 above plz @rushatgabhane

**Comment by rushatgabhane** [comment]
Similar here, the solution by @ShridharGoel  is different from @marufsharifi's PR. It also had anti patterns like platform specific code in files

So I don't think it is eligible for comp 😅
